src/main/java/zd/zero/waifu/motivator/plugin/motivation/BaseMotivation.kt

```diff
@@ -25,8 +25,20 @@ abstract class BaseMotivation(
 
     override fun motivate() {
         if (isAlertEnabled && isDistractionAllowed) {
-            if (isDisplayNotificationEnabled) displayNotification()
+            if (isDisplayNotificationEnabled) {
+                displayNotification()
+            } else {
+                useListener { it.onNotDisplayed() }
+            }
             if (isSoundAlertEnabled) soundAlert()
+        } else {
+            useListener { it.onNotDisplayed() }
```

Comment:
For L#31 and L#35, if we're using the same impl from 
`IdlePersonalityCore.ktL#33-L#39`,  if we can use it with `useListener { it. onDismissed() }`

Reply:
I think this was addressed by https://github.com/zd-zero/waifu-motivator-plugin/pull/183/commits/73a3f97dab8baaa2d8dc4576d0780c3ff4fa5203 , right?

Reply:
Yep, so apparently I commented on the same topic 😅